src/compiler/scala/tools/nsc/ast/parser/MarkupParsers.scala

```diff
@@ -172,20 +173,19 @@ trait MarkupParsers {
     }
 
     def appendText(pos: Position, ts: Buffer[Tree], txt: String): Unit = {
```

Comment:
There's a small change in behavior: if `preserveWS && txt.isEmpty`, before there was a call `append(txt)`, now not anymore. Is that intended? Is it significant?